apps/extract-stack/src/stack.ts

```diff
@@ -25,27 +30,37 @@ export function ExtractStack({ stack }: StackContext) {
       }
     },
   });
-
+  const mergeRequestQueue = new Queue(stack, "MRQueue")
```

Comment:
@ivke995 @dejan-crocoder @Ante-Koceic Let's figure out how to name resources properly!